pkg/e2e/dns/dns.go

```diff
@@ -45,7 +45,6 @@ func (kd *KubeDNS) Start(name string, args ...string) {
 
 	args = append(
 		args,
-		"--logtostderr",
```

Comment:
I'm wondering why it should be removed now if this is a kubedns option and the present PR does not modify kubedns code?

Reply:
Its a deprecated option in the binary and was causing the test to fail locally 
`
      --logtostderr                      log to standard error instead of files (default true) (DEPRECATED: will be removed in a future release, see https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
`

Reply:
I see, thanks!